package.json

```diff
@@ -1,75 +1,79 @@
 {
-  "name": "react-list-select",
-  "version": "0.4.0",
-  "license": "MIT",
-  "description": "List with selectable and navigable items",
-  "author": "Navaru",
-  "repository": "hawkrives/react-list-select",
-  "contributors": [
-    "Hawken Rives <hawkrives@gmail.com>"
-  ],
-  "keywords": [
-    "react-component"
-  ],
-  "main": "dist/index.js",
-  "files": [
-    "dist/",
-    "CHANGELOG.md"
-  ],
-  "scripts": {
-    "build": "babel ./src --out-dir ./dist",
-    "deploy": "gh-pages-deploy",
-    "flow": "flow",
-    "lint": "eslint --cache *.js example/*.js",
-    "example": "browserify -t [babelify] -e example/index.js > example/bundle.js",
-    "prepublishOnly": "npm run build",
-    "prettify": "prettier --write src/*.js example/*.{css,js}",
-    "post-deploy": "npm run clean"
-  },
-  "gh-pages-deploy": {
-    "staticpath": "example",
-    "prep": [
-      "example"
-    ],
-    "noprompt": true
-  },
-  "babel": {
-    "presets": [
-      "es2015",
-      "react"
-    ],
-    "plugins": [
-      "transform-class-properties"
-    ]
-  },
-  "prettier": {
-    "semi": false,
-    "singleQuote": true,
-    "tabWidth": 4,
-    "useTabs": true,
-    "bracketSpacing": false,
-    "trailingComma": "all"
-  },
-  "dependencies": {
-    "classnames": "^2.2.5",
-    "lodash": "^4.17.5"
-  },
-  "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^8.2.2",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babelify": "^8.0.0",
-    "browserify": "^16.1.1",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-flowtype": "^2.46.1",
-    "eslint-plugin-react": "^7.7.0",
-    "flow-bin": "^0.68.0",
-    "gh-pages-deploy": "^0.4.2",
-    "prettier": "^1.11.1",
-    "react": ">=15"
-  }
+	"name": "react-list-select-extended",
+	"version": "0.5.0",
+	"license": "MIT",
+	"description": "List with selectable and navigable items",
+	"author": "GersonDias",
+	"repository": "gersondias/react-list-select",
+	"contributors": [
+		"Hawken Rives <hawkrives@gmail.com>",
+		"Gerson Dias <eu@gersondias.net>"
+	],
+	"keywords": [
+		"react-component"
+	],
+	"main": "dist/index.js",
+	"files": [
+		"dist/",
+		"CHANGELOG.md"
+	],
+	"scripts": {
+		"build": "babel ./src --out-dir ./dist",
+		"deploy": "gh-pages-deploy",
+		"flow": "flow",
+		"lint": "eslint --cache *.js example/*.js",
+		"example": "browserify -t [babelify] -e example/index.js > example/bundle.js",
+		"prepublishOnly": "npm run build",
+		"prettify": "prettier --write src/*.js example/*.{css,js}",
+		"post-deploy": "npm run clean",
+		"start": "node node_modules/nodemon/bin/nodemon --watch ./src --exec 'npm run example && node_modules/http-server/bin/http-server ./example'"
```

Comment:
`nodemon ...` – npm automatically puts installed binaries into $PATH when you're running a script.